modules/manage/pages/cluster-maintenance/compaction-tuning.adoc

```diff
@@ -0,0 +1,48 @@
+= Optimize Compaction Settings
+:description: Redpanda's approach to compaction and options for configuring it
+
+Configure compaction for your cluster to optimize storage utilization.
+
+== Repanda Compaction Overview
+
+Compaction is an optional mechanism intended to reduce the storage needs of Redpanda topics. Compaction is enabled through configuration of a cluster or topic's cleanup policy. When compaction is enabled as part the cleanup policy, a background process executes on a pre-set interval to perform compaction operations. When triggered for a partition the process purges older versions of messages for a given key and only retains the most chronologically recent message in that partition. This is done by analyzing closed segments in the partition, copying the most recent messages for each key into a new segment,then deleting the source segments.
+
+image::shared:compaction-example.png[Example of topic compaction]
+
+This diagram provides an illustration of a compacted topic. Imagine a remote sensor network that uses image recognition to track appearances of red pandas in an area and devices send messages to a topic when they detect one. You might enable compaction to reduce topic storage while still maintaining a record in the topic of the last time each device saw a red panda, perhaps to see if they stop frequenting a given area. The left side of the diagram shows all messages sent across the topic. The right side illustrates the results of compaction; older messages for certain keys are deleted from the message log.
+
+There are two important considerations for compaction:
+
+*  When using tiered storage, compaction functions at the local level before segments are uploaded to storage. Once a segment is uploaded to storage it is not retrieved for further compaction operations. A key may therefore appear in multiple segments between tiered and local storage. 
+* Compaction on Redpanda is bounded by raft leadership terms. The same key may appear across multiple raft terms, as shown in the preceding diagram. 
+
+While compaction reduces storage needs, Redpanda's compaction (just like Kafka's) does not guarantee perfect de-duplication of a topic. Duplicates of a message, or additional messages, may still exist within a topic ordered by time. Compaction is not a complete topic operation; rather, it represents a best effort mechanism to reduce storage needs for topics created within a certain time window.
```

Comment:
>  ordered by time

I think you can remove this. Duplicates exists or they don't, independent of ordering.